app/templates/base.html

```diff
@@ -193,27 +193,27 @@ <h1>SOCIAL MEDIA</h1>
                                     <li>
                                         <img class="social-media-logo"
                                              src="{{ url_for('static', filename='img/facebook_logo.png') }}">
-                                        <a href="https://www.facebook.com/nycrecords"> Facebook</a>
+                                        <a href="https://www.facebook.com/nycrecords" target="_blank"> Facebook</a>
```

Comment:
For security, you should also use `rel=noopener` whenever you use `target="_blank"`